pkg/upstream/fetch.go

```diff
@@ -140,6 +140,10 @@ func pickEmbeddedClusterArtifacts(fetchOptions *types.FetchOptions) *kotsv1beta1
 			Charts:      imageutil.NewEmbeddedClusterOCIArtifactPath(fetchOptions.Airgap.Spec.EmbeddedClusterArtifacts.Charts, opts).String(),
 			ImagesAmd64: imageutil.NewEmbeddedClusterOCIArtifactPath(fetchOptions.Airgap.Spec.EmbeddedClusterArtifacts.ImagesAmd64, opts).String(),
 			Metadata:    imageutil.NewEmbeddedClusterOCIArtifactPath(fetchOptions.Airgap.Spec.EmbeddedClusterArtifacts.Metadata, opts).String(),
+			AdditionalArtifacts: map[string]string{
```

Comment:
can you please update the corrosponding pull test as well? these are the locations that need to be updated (you can find them if you search for `imagesAmd64` as well):
- https://github.com/replicatedhq/kots/blob/d2f589641ddfb4bb8bd143aff996a25815c3388f/pkg/tests/pull/cases/airgap/upstream/userdata/installation.yaml
- https://github.com/replicatedhq/kots/blob/d2f589641ddfb4bb8bd143aff996a25815c3388f/pkg/tests/pull/cases/airgap/wantResults/upstream/userdata/installation.yaml
- https://github.com/replicatedhq/kots/blob/d2f589641ddfb4bb8bd143aff996a25815c3388f/pkg/tests/pull/cases/airgap/wantResults/kotsKinds/userdata/installation.yaml